src/index.js

```diff
@@ -131,8 +131,8 @@ class Service {
 
     let { query, options } = multiOptions(id, params, this.id);
 
-    // We can not update the id
-    delete data[this.id];
```

Comment:
So my thought was we would only need:

```js
if (this.id === '_id') {
  delete data[this.id];
}
```

and the custom `id` field that is meant to be preserved needs to be passed. It just shouldn't be removed.